### ISSUE TITLE
Zikula console upgrade issue

### ISSUE THREAD

**Issue by MrMontesa** [zikula/core#3142]
| Q                 | A
| ----------------- | ---
| Zikula Version    | 1.4.3
| PHP Version       | 5.6.25

### Expected behavior
Proper update via console

### Actual behavior
php app/console zikula:upgrade spits error:

```
  [RuntimeException]                                                    
  Invalid data provided: ERROR: Error! There was a problem logging in.
```

### Steps to reproduce
This might be an issue related to my environment since I was able to properly update a plain 1.4.3 install to plain 1.4.4. Then I proceeded with the upgrade of my real testbed, where I encountered this error.
I was not able to get past this error. Also tried creating a new admin user with an easy password, but did not work.

Can someone explain at which point this fails? Also verbose output did not help my to find the issue. From my analyis I found the eror is a catch for 

```
try {
            $user = $this->databaseConnection->fetchAssoc('SELECT uid, pass FROM users WHERE uname= ?', [$object['username']]);
```
in `lib/Zikula/Bundle/CoreInstallerBundle/Validator/Constraints/AuthenticateAdminLoginValidator.php`
But I'm not too sure if the database connection itself was already properly made and the DB query fails.

I can properly login to the installation with either admin user. Any help to get this sorted is appreciated.
Thanks

**Comment by craigh** [comment]
> But I'm not too sure if the database connection itself was already properly made and the DB query fails.

It would only throw an exception if the DB connection fails. It should return empty `[]` if the user is not found.

since this is functioning correctly via CLI at travis (https://travis-ci.org/zikula/core) it would seem your issue is with your environment (as you said) or something else.

If you can provide a reproducible behavior, please reopen.

**Comment by MrMontesa** [comment]
A little bit more supportive answer would have been great as this is a persistant issue in my setup. 
How to troubleshoot in more detail?
How to get more debug info?

There was an error loggin in: I know for sure that login credentials provided are 100% correct. You cnfirmed that the user was pulled correctly out of the database. Else it would have spit empty message.

So Whats the login problem? I'm puzzled.
Thanks for your help

**Comment by craigh** [comment]
Did you update the db credentials in `custom_parameters.yml` as outlined in the [docs](https://github.com/zikula/core/blob/1.4/UPGRADE-1.4.md#if-upgrading-from-core-14x)?

> You cnfirmed that the user was pulled correctly out of the database.

I did not. I said it would only throw an exception if the DB connection fails. The exception means there is a problem connecting to the DB - not with your site admin credentials, but with your DB connection settings in the `custom_parameters.yml` file.

**Comment by craigh** [comment]
I may have found your error

>   Invalid data provided: ERROR: Error! There was a problem logging in.

stay tuned.

**Comment by MrMontesa** [comment]
>Did you update the db credentials in custom_parameters.yml as outlined in the docs?

I think the docs suggest to NOT touch content of the file, but copy file as is, right.
This is how I du updates.

```
~/doc_root/zk143/
cd ~/doc_root
wget 1.44.zip
unzip 1.44.zip
mv Zikula zk144
cp zk143/config/personal_config zk144/config/
cp zk143/app/config/custom_parameters.yml zk144/app/config/
cd zk144
php ./app/console zikula:upgrade
```

> stay tuned

Sounds great!

**Comment by rallek** [comment]
@MrMontesa why do you not use the zk144.tar.gz file? Than all file permissions are fine if you unpack it with tar -xpzf (p is important for keeping file permissions)

**Comment by craigh** [comment]
closed by #3173
